### PR TITLE
Docs: Added meta-reducer to injected feature config example.

### LINKED ISSUE #2019: Docs: Improve example for dependency injection.
## Other information:
My recent work has taken me into dependency injection. The `getConfig` used class in section [Dependency Injection -> Injecting Feature Config](https://ngrx.io/guide/store/recipes/injecting#injecting-feature-config) could be expanded to have more information about the object.

Current:

```typescript
export function getConfig(someService: SomeService): StoreConfig<fromFeature.State> {
  // return the config synchronously.
  return {initialState: someService.getInitialState()};
}
```

Proposed:

```typescript
export function getConfig(someService: SomeService): StoreConfig<fromFeature.State> {
  return {
    initialState: someService.getInitialState(),

    reducerFactory(reducerMap, initialState) {
      return createReducer(
        initialState,

        on(fromFeature.FeatureAction, (state, { payload }) => ({
          ...state,
          foo: someService.reducerAction(payload)
        ))
      );
    },

    metaReducers: [
      loggerReducerFactory(someService.loggerConfig())
    ]
  };
}
```

## I would be willing to submit a PR for the docs :heart:

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

### PR DESCRIPTION
**Pull request by bhalash**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

* The [Dependency Injection -> Injecting Feature Config](https://ngrx.io/guide/store/recipes/injecting#injecting-feature-config) example doesn't have information on using a meta-reducer factory.
* The DI guide refers to `META_REDUCERS`, whereas [V8 Upgrade Guide -> `META_REDUCERS` token](https://ngrx.io/guide/migration/v8#meta_reducers-token) refers to `USER_PROVIDED_META_REDUCERS`

Closes #2019 

## What is the new behavior?

I've updated and expanded examples.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by timdeschryver** [comment]
I'm not sure why the test step is failing, I've already restarted job twice

**Comment by bhalash** [comment]
Mine is an evil laugh. No idea why it'd be failing unfortunately, except maybe a cached build?

**Comment by bhalash** [comment]
@timdeschryver I noticed a further omission in the same document that I've added on this PR.

File: projects/ngrx.io/content/guide/store/recipes/injecting.md [hunk 1]
```diff
@@ -56,12 +56,12 @@ export class FeatureModule {}
 
 ## Injecting Meta-Reducers
 
-To inject 'middleware' meta reducers, use the `META_REDUCERS` injection token exported in
+To inject 'middleware' meta reducers, use the `USER_PROVIDED_META_REDUCERS` injection token exported in
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store/recipes/injecting.md]: We want to keep this to `META_REDUCERS`.

The `META_REDUCERS` token provides a way for library owners to add meta-reducers.

Comment (bhalash) [reply projects/ngrx.io/content/guide/store/recipes/injecting.md]: @timdeschryver Oh cool, understood, I'll resolve this!

Comment (bhalash) [reply projects/ngrx.io/content/guide/store/recipes/injecting.md]: @timdeschryver I have rolled back my change per what you've said.

**Pull request merged**